python-lib/tagger.py

```diff
@@ -1,9 +1,110 @@
+"""Module to prepare the tagging of the documents
+splits on sentences and instanciates a Matcher to tag the documents"""
+
+from spacy_tokenizer import MultilingualTokenizer
```

Comment:
Another overall comment: can you add a lot more logging to your methods? You can use the text preparation plugin as example